queries.sql

```diff
@@ -11,6 +11,44 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg  BETWEEN 10.4 AND 17.3;
 
+BEGIN; 
 
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
 
+SAVEPOINT my_first_savepoint;
+
+UPDATE animals 
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT my_first_savepoint;
+
+UPDATE animals 
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+
+SELECT count(*) AS no_of_animals
+FROM animals;
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️

Reply:
@HAMMAS-SALEEM  I didn't get you. Could you please tell me in detail?

Reply:
I am just suggesting you to make a logo of your own and put that into your readme files 👍